tools/run_tests/run_xds_tests.py

```diff
@@ -1106,6 +1107,7 @@ def __init__(self, compute, project):
             result.state = 'PASSED'
             result.returncode = 0
         except Exception as e:
+            logger.error('Test case %s failed: %s' % (test_case, e))
```

Comment:
Loggers are recommended to be used life `printf`.
```Python
logger.error('Test case %s failed: %s', test_case, e)
```

Reply:
Done